src/components/LibraryCardForm/LibraryCardForm.tsx

```diff
@@ -350,7 +350,7 @@ const LibraryCardForm = () => {
           getValues={getValues}
           register={register}
           errors={errors}
-          errorMessages={errorMessages}
+          errorMessage={errorMessages.username}
```

Comment:
Why this change?

Reply:
It seems unnecessary to send the whole object that can have multiple and unrelated properties. So instead, just the error message string for the username is passed.